lab_1/unk.json

```diff
@@ -0,0 +1 @@
+{"name": "unk", "freq": {"at": 1, "first": 1, "von": 1, "frisch": 1, "thought": 1, "the": 15, "bees": 3, "were": 2, "responding": 2, "only": 2, "to": 4, "scent": 1, "of": 5, "foodbut": 1, "what": 1, "did": 2, "third": 1, "dance": 2, "mean": 1, "and": 3, "if": 2, "scenthow": 1, "could": 1, "they": 1, "also": 2, "sniff": 1, "down": 1, "food": 1, "hundreds": 1, "metres": 4, "away": 4, "from": 1, "hive": 1, "foodwhich": 1, "was": 4, "sometimes": 1, "downwind": 1, "on": 1, "a": 2, "hunch": 1, "he": 5, "started": 2, "gradually": 1, "moving": 1, "thefeeding": 1, "dish": 4, "further": 2, "noticed": 1, "as": 1, "so": 1, "that": 1, "dances": 1, "thereturning": 1, "scout": 1, "changing": 1, "placed": 1, "feeding": 2, "over": 1, "ninemetres": 1, "second": 1, "type": 1, "sickle": 1, "version": 1, "came": 1, "into": 1, "playbut": 1, "once": 1, "moved": 2, "it": 1, "past": 1, "scouts": 1, "would": 1, "then": 1, "start": 1, "dancing": 1, "thirdquite": 1, "different": 1, "waggle": 1, "dancethe": 1, "measurement": 1, "actual": 1, "distance": 1, "too": 1, "concluded": 1, "precise": 1, "forexample": 1, "indicated": 1, "by": 1, "complete": 1, "runsthrough": 1, "pattern": 1, "in": 1, "seconds": 1, "when": 1, "awaythe": 1, "number": 1, "dropped": 1, "eleven": 1}, "n_words": 98}
```

Comment:
remove this file

Reply:
But it's needed to compare text in unknown language with other profiles and to detect the language...